pandas/core/indexes/category.py

```diff
@@ -300,6 +301,12 @@ def itemsize(self):
         # Size of the items in categories, not codes.
         return self.values.itemsize
 
+    def _wrap_setop_result(self, other, result):
+        name = get_op_result_name(self, other)
```

Comment:
use _shallow_copy here, we *never* call _simple_new directly.  (another reason why the super method should use ``_shallow_copy``, but that's for new PR)

Reply:
done

Reply:
can you move this method to Index.base? does it break anything?